src/main/java/org/symphonyoss/symphony/messageml/util/IDataProvider.java

```diff
@@ -43,6 +43,12 @@
    */
   IUserPresentation getUserPresentation(Long uid) throws InvalidInputException;
 
+  /**
+   * Retrieve user information of the author of the message
+   * @return User presentation object
+   */
+  IUserPresentation getAuthorPresentation() throws InvalidInputException;
```

Comment:
I don't understand why we need this method. The implementation should just call _getUserPresentation_ with the desired ID or email.